### ISSUE TITLE
Eviction warning on scrooge core

### ISSUE THREAD

**Issue by RayRoestenburg** [CommBank/uniform#39]
```
[warn] There may be incompatibilities among your library dependencies.
[warn] Here are some of the libraries that were evicted:
[warn] 	* com.twitter:scrooge-core_2.10:3.6.0 -> 3.14.1
[warn] Run 'evicted' to see detailed eviction warnings
```
in uniform dependencies:
```
def scrooge      = "3.14.1"
def bijection    = "0.6.3"
...
"com.twitter"              %% "scrooge-core"                  % scrooge,
"com.twitter"              %% "bijection-scrooge"             % bijection
```
bijection-scrooge `0.6.3` depends on scrooge-serializer_2.10 `3.6.0` depends on scrooge-runtime_2.10 `3.6.0` depends on scrooge-core_2.10 `3.6.0`.

scrooge-core_2.10 3.6.0 should be excluded in bijection scrooge dependency.

**Comment by stephanh** [comment]
@SamRoberts is that still an issue?

**Comment by SamRoberts** [comment]
This has been fixed by #42
